common/koa-middleware/withToggles.js

```diff
@@ -33,16 +39,42 @@ const parseCookies = function(req) {
 function withToggles(ctx, next) {
   const cookies = parseCookies(ctx.req);
   const togglesCookies = cookies.filter(cookie =>
-    cookie.key.startsWith('toggle_')
+    cookie.key.startsWith(cookiePrefix)
   );
   const toggles = togglesCookies.reduce((acc, cookie) => {
     return Object.assign({}, acc, {
-      [cookie.key.replace('toggle_', '')]: cookie.value === 'true',
+      [cookie.key.replace(cookiePrefix, '')]: cookie.value === 'true',
     });
   }, {});
 
   ctx.toggles = { ...defaultToggleValues, ...toggles };
+  enableDisableToggler(ctx);
   return next();
 }
 
+function enableDisableToggler(ctx) {
+  if (ctx.toggles) {
+    // enable toggler of feature
```

Comment:
We could call this a `featureToggle`, [which is common language for it](https://martinfowler.com/articles/feature-toggles.html).